pages/common/pssh.md

```diff
@@ -20,7 +20,7 @@
 
 - Run a command with extra SSH arguments:
 
-`pssh -i -h {{path/to/hosts_file}} -x "-O StrictHostKeyChecking=no" date`
+`pssh -i -h {{path/to/hosts_file}} -x "{{extra_ssh_args}}" date`
```

Comment:
Ah, the concrete example here like `-O SomeOption=foobar` was great - it's not intuitive without it. I was just questioning the inclusion of an option here by default that reduces security haha

Perhaps something like `"{{-O VisualHostKey=yes}}"` would be more suited:question: